include/xrpl/protocol/TxFlags.h

```diff
@@ -214,6 +216,15 @@ constexpr std::uint32_t tfAMMClawbackMask = ~(tfUniversal | tfClawTwoAssets);
 // BridgeModify flags:
 constexpr std::uint32_t tfClearAccountCreateAmount     = 0x00010000;
 constexpr std::uint32_t tfBridgeModifyMask = ~(tfUniversal | tfClearAccountCreateAmount);
+
+// Batch Flags:
+constexpr std::uint32_t tfAllOrNothing                 = 0x00010000;
+constexpr std::uint32_t tfOnlyOne                      = 0x00020000;
+constexpr std::uint32_t tfUntilFailure                 = 0x00040000;
+constexpr std::uint32_t tfIndependent                  = 0x00080000;
+constexpr std::uint32_t const tfBatchMask =
+    ~(tfUniversal | tfAllOrNothing | tfOnlyOne | tfUntilFailure | tfIndependent | tfInnerBatchTxn);
```

Comment:
I don't think that's correct. `tfInnerBatchTxn`should be outside the negation.
```suggestion
constexpr std::uint32_t const tfBatchMask =
    ~(tfUniversal | tfAllOrNothing | tfOnlyOne | tfUntilFailure | tfIndependent) | tfInnerBatchTxn;
```

Example:
`~(tfUniversal | tfAllOrNothing | tfOnlyOne | tfUntilFailure | tfIndependent) ` gives binary value of `00111111111100001111111111111111`, where a 0 bit is an acceptable flag.

now, we need to `| tfInnerBatchTxn` in order to flip the second MSB to 1, into `01111111111100001111111111111111`

Reply:
fixed: https://github.com/XRPLF/rippled/pull/5060/commits/b7b6b8a92f646bc377ad9fba3b35d145e7b7ef3a